docs/01-start-here/06-testing-tips.md

```diff
@@ -0,0 +1,157 @@
+  This is especially important if your feature is very resource heavy. For reasons unknown, but possibly connected to the way safari manages memory, iOS devices have proved more susceptable
+   Good examples of old devices to test on are iPhone 4S, iPad3 (both known to be prone to crashes) and the Samsung Galaxy S3.
+    [Interesting page talking about iOS crashes](http://stackoverflow.com/questions/22039534/ios-browser-crashes-due-to-low-memory)
+
+- Devices with strange screen sizes. If you re interested in layout and rendering for your change, the iphone 5S is a popular device with an abnormally narrow screen,
+    the Galaxy S4 mini is a miniaturised version of the GalaxyS4 with a very small screen. iPad Mini and Galaxy Note also have interesting dimensions and have shown layout issues from time to time
+     -especially on fronts
+
+- Rotate your device - another layout trick that is always worth doing. Load the page and rotate from portrait to landscape.
+   Refresh and rotate from landscape to portrait. This is a common cause of redrawing in the real world that can often show up issues.
+    A good device to test this on is the Nexus7 - this is because the portrait and landscape views on this device correspond to different screen-width breakpoints.
+
+- Test on AMP! [Google are making amp the default in mobile search results](http://searchengineland.com/amp-live-in-google-259109).
+Currently there are lots of dependencies and many changes that don't seem to be directly related to AMP will have an impact on the display of amp pages.
+
+### Testing AMP Pages on local build
+Whenever you are making a significant frontend change - especially one that has any impact on any templates
+you should be sure to check the display of an amp page to make sure it looks alright.
+
+When running frontend locally, the AMP version of pages can be viewed by adding the `?amp` querystring to the end of the URL. For example:
+
+```
+http://localhost:9000/world/2015/oct/15/obama-delay-withdrawal-us-troops-afghanistan?amp
+```
+
+### Amp validation on local
+Amp pages must pass validation in order to be displayed in Google search results.
+If an Amp page fails validation it is highly unlikely it will ever be served to a user.
+It is important to test Amp pages pass validation as part of any checking for frontend changes.
+
+To test this on a page run from local, request an amp page as shown above, and add the fragment `#development=1` to the end of the URL. For example:
+
+```
+http://localhost:9000/world/2015/oct/15/obama-delay-withdrawal-us-troops-afghanistan?amp#development=1
+```
+
+Before committing any frontend changes, you should also run the Amp validation tests.
+This is done by first running dev-build or article project on your local machine.
+In a new terminal tab, navigate to the frontend project and run:
+
+```
+make validate-amp
+```
+
+### Testing Amp pages on the live website
```

Comment:
A good point! Amp is only available for articles and liveblogs. I've updated docs to this effect.